site/source/user-manual/alt-marketplaces.rst

```diff
@@ -4,14 +4,12 @@
 Alt Marketplaces
 ================
 
-It is critical to Start9's mission that Start9 NOT be a central point of failure.  The Marketplace is a critical piece of infrastructure and therefore EmbassyOS supports accessing alternative marketplaces.
+It is critical to Start9's mission that Start9 does **not** be a central point of failure. The Marketplace is a critical piece of infrastructure and therefore EmbassyOS supports accessing alternative marketplaces.
```

Comment:
does not be?  Not sure if grammatically incorrect, but sounds awkward

Perhaps "is not a central point..." or "does not become"